test/test_serialization.py

```diff
@@ -869,6 +869,9 @@ def __new__(cls, elem, *args, **kwargs):
         r.elem = elem
         return r
 
+    def clone(self):
+        return type(self)(self.elem.clone())
+
```

Comment:
@albanD just realized these tests were here - I guess they didn't catch the case this PR addresses because the tensors tested here are old-style wrapper tensors with meta tensors underneath. Should we get rid of these tests to avoid duplication? My preference is to keep tests for all semantics related to tensor subclasses together - wdyt? Alternatively, maybe leave them in for now and clean them up later when the more comprehensive tests land?